.github/ISSUE_TEMPLATE/invitation.md

```diff
@@ -7,10 +7,10 @@ labels: invite me to the organisation
 
 <!---
 Invitation will be sent for the GitHub Organization soon. We look forward to having you part of our community :nerd_face:
-Don't forget after accepting to make it public so it appears on your github profile for everyone else to see, you can do this by finding your name in the github organization list and change the dropdown to public https://github.com/orgs/EddieJaoudeCommunity/people
+Don't forget after accepting to make it public so it appears on your github profile for everyone else to see, you can do this by finding your name in the github organization list and change the dropdown to public https://github.com/orgs/EddieHubCommunity/people
 Tips for practicing:
 - Customise your GitHub profile, here is a live stream on it https://www.youtube.com/watch?v=cT6GXCuS0Zo
-- Practice repo, instructions how to add your name to the README in the README https://github.com/EddieJaoudeCommunity/hacktoberfest-practice
+- Practice repo, instructions how to add your name to the README in the README https://github.com/EddieHubCommunity/hacktoberfest-practice
```

Comment:
I suggest to change this line.

suggestion:
`Practice repo, instructions on how to add your name in the README https://github.com/EddieHubCommunity/hacktoberfest-practice`